src/vega_viewer/vega/specs/histogram.cljs

```diff
@@ -49,7 +53,8 @@
 
 (defn generate-histogram-chart-vega-spec
   [{values :data :keys [height status-text width]}
-   & {:keys [responsive? abbreviate-x-axis-tick-labels?]}]
+   & {:keys [responsive? abbreviate-x-axis-tick-labels?
+             x-axis-title y-axis-title custom-duration-chart?]}]
```

Comment:
I think it'd be better to name `custom-duration-chart?` more specifically, follow the pattern of `abbreviate-x-axis-tick-labels?` we could call it `x-axis-time-format-labels?`